perf/dev_exporter.go

```diff
@@ -0,0 +1,160 @@
+package perf
+
+// based on "go.opencensus.io/examples/exporter"
+
+import (
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"log"
+	"regexp"
+	"sync"
+	"time"
+
+	"go.opencensus.io/metric/metricdata"
+	"go.opencensus.io/metric/metricexport"
+	"go.opencensus.io/trace"
+
+	"go.viam.com/utils"
+)
+
+// DevelopmentExporter exports metrics and span to log file.
+type DevelopmentExporter struct {
```

Comment:
Is this copy/paste from some other package? wondering how we can re-use centrally (can be handled later).

Reply:
yes, the comment below package says `based on "go.opencensus.io/examples/exporter"`. This makes some choices on it. Not sure what you mean by reuse centrally? This will be used in app.